stake-pool/js/src/index.ts

```diff
@@ -702,20 +702,22 @@ export async function increaseValidatorStake(
       stakePoolAddress,
       new BN(ephemeralStakeSeed),
     );
-    StakePoolInstruction.increaseAdditionalValidatorStake({
-      stakePool: stakePoolAddress,
-      staker: stakePool.account.data.staker,
-      validatorList: stakePool.account.data.validatorList,
-      reserveStake: stakePool.account.data.reserveStake,
-      transientStakeSeed: transientStakeSeed.toNumber(),
-      withdrawAuthority,
-      transientStake,
-      validatorStake,
-      validatorVote,
-      lamports,
-      ephemeralStake,
-      ephemeralStakeSeed,
-    });
+    instructions.push(
+      StakePoolInstruction.increaseAdditionalValidatorStake({
+        stakePool: stakePoolAddress,
+        staker: stakePool.account.data.staker,
+        validatorList: stakePool.account.data.validatorList,
+        reserveStake: stakePool.account.data.reserveStake,
+        transientStakeSeed: transientStakeSeed.toNumber(),
+        withdrawAuthority,
+        transientStake,
+        validatorStake,
+        validatorVote,
+        lamports,
+        ephemeralStake,
+        ephemeralStakeSeed,
+      }),
+    );
```

Comment:
lol oops, thanks for noticing and fixing this. As @buffalojoec pointed out, can you move this fix to a separate PR?